packages/postgres-database/requirements/prod.txt

```diff
@@ -8,4 +8,5 @@
 --requirement _base.txt
 --requirement _migration.txt
 
+simcore-common-library @ ../common-library/
```

Comment:
what about dev.txt and ci.txt?

Reply:
True

Reply:
but in any case the `migration` service is always used in production even the image tagged as dev